source/styles/main.styl

```diff
@@ -116,14 +116,14 @@ blockquote
     line-height 1.2rem
 
 nav
-  font-size 0.9rem
+  font-size 0.8rem
   margin 70px 0 30px 0
   transition all 0.2s
 
   a
     display inline-block
     line-height 2.2
-    padding 0 15px
+    padding 0 10px
```

Comment:
Ya although I made this change, I had mixed feelings. (I had a nice internal debate before landing it.)

As we keep adding pages (soon) we'll need to really re-evaluate nav. Maybe a column-based "jumbo footer" or something? Also the top-nav maybe will need an extra section-or-two, once we have sections.